wand/gui.py

```diff
@@ -51,33 +51,33 @@ def __init__(self, display_name, gui):
         self.osa.showGrid(y=True)
         self.osa_curve = self.osa.plot(pen='y', color=self.colour)
 
-        self.fast_mode = QtGui.QCheckBox("Fast mode")
-        self.auto_exposure = QtGui.QCheckBox("Auto expose")
+        self.fast_mode = QtWidgets.QCheckBox("Fast mode")
+        self.auto_exposure = QtWidgets.QCheckBox("Auto expose")
 
-        self.exposure = [QtGui.QSpinBox() for _ in range(2)]
+        self.exposure = [QtWidgets.QSpinBox() for _ in range(2)]
         for idx in range(2):
             self.exposure[idx].setSuffix(" ms")
             self.exposure[idx].setRange(0, 0)
 
-        self.laser_status = QtGui.QLineEdit()
+        self.laser_status = QtWidgets.QLineEdit()
         self.laser_status.setReadOnly(True)
 
-        self.f_ref = QtGui.QDoubleSpinBox()
+        self.f_ref = QtWidgets.QDoubleSpinBox()
         self.f_ref.setSuffix(" THz")
         self.f_ref.setDecimals(7)
         self.f_ref.setSingleStep(1e-6)
         self.f_ref.setRange(0., 1000.)
 
         # context menu
-        self.menu = QtGui.QMenu()
-        self.ref_editable = QtGui.QAction("Enable reference changes",
+        self.menu = QtWidgets.QMenu()
+        self.ref_editable = QtWidgets.QAction("Enable reference changes",
                                           self.dock)
         self.ref_editable.setCheckable(True)
         self.menu.addAction(self.ref_editable)
 
         for label in [self.detuning, self.name, self.frequency, self.f_ref]:
             label.contextMenuEvent = lambda ev: self.menu.popup(
-                QtGui.QCursor.pos())
+                QtWidgets.QCursor.pos())
```

Comment:
Are you sure about this? I had to revert this line to make right-click context menus appear.